### PR TITLE
ARROW-4632: [Ruby] Add BigDecimal#to_arrow

### PR DESCRIPTION
**Pull request by mrkn**

### CODE REVIEW

File: ruby/red-arrow/test/test-bigdecimal.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+class BigDecimalTest < Test::Unit::TestCase
+  test("to_arrow") do
```
Comment (kou) [inline ruby/red-arrow/test/test-bigdecimal.rb]: Could you use `"#to_arrow"` like other tests do.

File: ruby/red-arrow/test/test-bigdecimal.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    arrow_decimal = BigDecimal("3.14").to_arrow
```
Comment (kou) [inline ruby/red-arrow/test/test-bigdecimal.rb]: Aligning compare targets will be more readable:

```suggestion
    assert_equal(Arrow::Decimal128.new("3.14"),
                 BigDecimal("3.14").to_arrow)
```

**Review by kou — APPROVED** [review]
+1
Could you fix trivial problems? Then I'll merge this.

File: ruby/red-arrow/test/test-bigdecimal.rb (hunk 1 above)
Comment (mrkn) [reply ruby/red-arrow/test/test-bigdecimal.rb]: Fixed.

File: ruby/red-arrow/test/test-bigdecimal.rb (hunk 2 above)
Comment (mrkn) [reply ruby/red-arrow/test/test-bigdecimal.rb]: Fixed.

**Comment by kou** [comment]
CI is passed. I'll merge this.

**Pull request closed without merging**